packages/@o3r/core/schematics/library/index.spec.ts

```diff
@@ -23,26 +23,40 @@ describe('New module generator', () => {
   });
 
   it('should generate the minimum mandatory files', async () => {
+    initialTree.create('angular.json', '{"version": 1, "projects": {} }');
+    initialTree.create('package.json', '{ "version": "0.0.0-test" }');
     const runner = new SchematicTestRunner('schematics', collectionPath);
-    const tree = await runner.runExternalSchematic('schematics', 'module', {
+    const angularPackageJson = require.resolve('@schematics/angular/package.json');
+    runner.registerCollection('@schematics/angular', path.resolve(path.dirname(angularPackageJson), require(angularPackageJson).schematics));
+    const tree = await runner.runExternalSchematic('schematics', 'library', {
       path: 'packages-test',
       name: '@my/new-module',
       skipLinter: true
     }, initialTree);
 
     expect(tree.readContent('/packages-test/@my/new-module/package.json')).toContain('"name": "@my/new-module"');
     expect(tree.exists('/packages-test/@my/new-module/collection.json')).toBe(true);
+    expect(tree.exists('/packages-test/@my/new-module/schematics/ng-add/schema.json')).toBe(true);
     expect(tree.exists('/packages-test/@my/new-module/project.json')).toBe(false);
     expect(JSON.parse(tree.readContent('/tsconfig.base.json')).compilerOptions.paths['@my/new-module']).toContain('packages-test/@my/new-module/src/public_api');
     expect(JSON.parse(tree.readContent('/tsconfig.build.json')).compilerOptions.paths['@my/new-module'][0]).toBe('packages-test/@my/new-module/dist');
     expect(tree.files.length).toBeGreaterThan(16);
   });
 
-  describe('in NX monorepo', () => {
+  // Skip relative to nx generator issue
+  // TODO: re-enable when Nx Alias conflict is fixed
```

Comment:
we can mention https://github.com/AmadeusITGroup/otter/issues/491 here

Reply:
Not related to this issue.
The problem is that instead of calling the `library` schematic of `@anguler/schamtics`, Nx is calling its alias `lib`.
`lib` is an alias a different schematics which result to an error.
I did not put links to issue here because I did not find any issue open on there side.